openapi.yaml

```diff
@@ -0,0 +1,149 @@
+        high: { type: number, format: double, description: "Rolling 24-hours highest transaction price" }
+        low: { type: number, format: double, description: "Rolling 24-hours lowest transaction price" }
+    Tickers:
+      type: array
+      items: {$ref: '#/components/schemas/Ticker'}
+    Order:
+      type: object
+      properties:
+        ticker_id: { type: string, description: "Identifier of a ticker with delimiter to separate base/target, eg. ADA/GENS" }
+        timestamp: { type: string, description: "" }
+        bids: { type: array, items: { type: array, items: { type: number, minItems: 2, maxItems: 2, description: "An array containing 2 elements. The offer price and quantity for the bid order" }}}
+        asks: { type: array, items: { type: array, items: { type: number, minItems: 2, maxItems: 2, description: "An array containing 2 elements. The ask price and quantity for the ask order" }},}
+      example:
+        {
+          "ticker_id" : "ADA/GENS",
+          "timestamp" : "1700050000",
+          "bids" : [[ "49.8", "0.50000000" ], [ "49.9", "6.40000000" ]],
+          "asks" : [[ "50.1", "9.20000000" ], [ "50.2", "7.90000000" ]]
+        }
+    Orders:
+      type: array
+      items: {$ref: '#/components/schemas/Order'}
+    Trade:
+      type: object
+      required: [ticker_id, base_currency, target_currency, pool_id, last_price, base_volume, target_volume, liquidity_in_usd]
+      properties:
+        trade_id: { type: string, description: "Identifier of a ticker with delimiter to separate base/target, eg. ADA/GENS" }
+        price: { type: number, format: double, description: "Transaction price of base asset in target currency." }
+        base_volume: { type: number, format: double, description: "Transaction amount in base pair volume." }
+        target_volume: { type: number, format: double, description: "Transaction amount in target pair volume." }
+        trade_timestamp: { type: integer, description: "Unix timestamp in milliseconds for when the transaction occurred." }
+        type: { type: string, description: "Used to determine the type of the transaction that was completed. Buy: Identifies an ask that was removed from the order book. Sell: Identifies a bid that was removed from the order book." }
+    Trades:
+      type: object
+      properties:
+        buy : { type: array, items: {$ref: '#/components/schemas/Trade' }}
+        sell : { type: array, items: {$ref: '#/components/schemas/Trade'}}
+      example:
+        {
+          "buy": [{ "trade_id":1234567, "price":"50.1", "base_volume":"0.1", "target_volume":"1", "trade_timestamp":"1700050000", "type":"buy" }],
+          "sell": [{ "trade_id":1234567, "price":"50.1", "base_volume":"0.1", "target_volume":"1", "trade_timestamp":"1700050000", "type":"sell" }]
+        }
+    Price:
+      type: object
+      properties:
+        coin_a_close: { type: number, format: double, description: Coin A close }
```

Comment:
Thie section is influenced by the Maestro API definition. We need to bring the naming in line.

Is it:
 - base and target?
 - Currency and commodity?
 - coin A and coin B?
 
 We should use a single wording in the whole API.